renderer/main.js

```diff
@@ -94,15 +95,18 @@ function onState (err, _state) {
   // ...window visibility state.
   document.addEventListener('webkitvisibilitychange', onVisibilityChange)
 
-  sound.play('STARTUP')
+  // Log uncaught JS errors
+  window.addEventListener('error', (err) => telemetry.logUncaughtError('window', err.error), true)
```

Comment:
ya the MDN docs say that you need to capture the error event to get all the errors. errors that happen on an img or video tag etc won't bubble up to the window automatically